content/docs/guides/custom-name-resolution.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Custom Name Resolution
+description: >-
+  Explains standard name resolution, the custom name resolver interface,
+  and how to write an implementation.
+---
+
+### Overview
+
+Name resolution is fundamentally about service discovery. When sending a request
+to `grpc.io`, the client must determine the IP address of that service name.
```

Comment:
Do we want to use `example.com` just to be clear we're not actually expecting to receive RPCs on our website? If not, we should be clear that this is just an example host.

Reply:
Example removed.